pacstall/__main__.py

```diff
@@ -27,43 +27,50 @@
 import sys
 from fcntl import LOCK_EX, LOCK_NB, lockf
 from getpass import getuser
+from logging import getLogger
 from time import sleep
 
 from rich.traceback import install
 
-from pacstall.api import message
+from pacstall.api import logger
 from pacstall.api.error_codes import ErrorCodes
 from pacstall.cmds import download
 from pacstall.parser import parse_arguments
 
 
-def main() -> None:
+def main() -> int:
     """Main Pacstall function."""
 
+    logger.setup_logger()
+    log = getLogger()
+
     install(
         show_locals=True
     )  # --> Install Rich's traceback handler for better looking tracebackes
 
     args = parse_arguments()
-    print(args)
+    log.debug(f"{args = }")
 
     if getuser() != "root":
-        message.fancy("error", "Pacstall needs to be launched as root!")
+        log.error("Pacstall needs to be launched as root")
         sys.exit(ErrorCodes.USAGE_ERROR)  # --> command line usage error
 
     if args.command in ["install", "remove", "upgrade"]:
         lock_file = open("/var/lock/pacstall.lock", "w")
         while True:
             try:
                 lockf(lock_file, LOCK_EX | LOCK_NB)
+                log.debug("Lock acquired")
                 break
             except OSError:
-                message.fancy("error", "Pacstall is already running another instance")
+                log.error("Pacstall is already running another instance")
```

Comment:
Shouldn't this be a warning instead, since pacstall will run when it's able to

Reply:
https://github.com/pacstall/pacstall/pull/471/commits/9b0857511ca0f2b729d0854e65f9c89c7700a7bf